indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+
+<div
+  class="usa-summary-box"
+  role="region"
+  aria-labelledby="summary-box-key-information"
+>
+  <div class="usa-summary-box__body">
+    <h3 class="usa-summary-box__heading" id="summary-box-key-information">
+      Supported Metadata
+    </h3>
+    <div class="usa-summary-box__text">
+      <ul class="usa-list">
+        <li>
+          <a class="usa-summary-box__link" href="#main-page-content">Main Content</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#title">Title</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#description">Description</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#tags--keywords">Tags</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#created-date">Created Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#changed-date">Changed Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#crawling-metadata">Crawling Metadata</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#thumbnail-image">Thumbnail Image</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#audience">Audience</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#content-type">Content Type</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#searchgov-custom-fields">Search.gov Custom Fields</a> 
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+### Fields by Type
+
+|[Standard Fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)|[OpenGraph Fields](https://ogp.me/)|[DublinCore Fields](https://www.dublincore.org/)|
+|:----|:----|:----|:----|
+|[`<main>`](#main-page-content)|[`og:title`](#title)|[`dcterms.keywords`](#tags--keywords)|
+|[`<title>`](#title)|[`og:description`](#description)|[`dc.subject`](#tags--keywords)|
+|[`description`](#description)|[`article:section`](#tags--keywords)|[`dcterms.subject`](#tags--keywords)|
+|[`keywords`](#tags--keywords)|[`article:tag`](#tags--keywords)|[`dcterms.audience`](#audience)|
+|[`robots`](#crawling-metadata)|[`article:published_time`](#created-date)|[`dc.type`](#content-type)|
+||[`article:modified_time`](#changed-date)|[`dcterms.type`](#content-type)|
+||[`og:image`](#thumbnail-image)|[`dc.date`](#created-date)|
+|||[`dc.date.created`](#created-date)|
+|||[`dcterms.created`](#created-date)|
+
+
+### Main Page Content
+We look for the following field:
+```
+<main>
+```
+- **Detail:** Allows the search engine to target the actual content of the page and avoid headers, sidebars and other page content not useful to search. [Read more about the <main> element here]({{ site.baseurl }}/indexing/how-search-engines-index-content-better-discoverability.html#main-element)
```

Comment:
Let's restructure this read more to "Read more about the [<main> element](link goes here)."

Reply:
Fixed